plugins/source/crowdstrike/codegen/recipes/incidents.go

```diff
@@ -11,6 +13,14 @@ func CrowdScore() []*Resource {
 			SubService: "crowdscore",
 			Struct:     &models.DomainEnvironmentScore{},
 			PKColumns:  []string{"id"},
+			SkipFields: []string{"Timestamp"},
```

Comment:
I know this is still in draft, but can we use a custom type transformer instead of re-defining the columns?
Example in https://github.com/cloudquery/cloudquery/blob/a59f2f26a50c997beeb52c4415a49ac589825f99/plugins/source/vercel/codegen/main.go#L51